tests/integration/apigateway/test_apigateway_api.py

```diff
@@ -73,6 +75,25 @@ def _factory(*args, **kwargs):
         delete_rest_api_retry(apigateway_client, rest_api_id)
 
 
+@pytest.fixture
+def apigw_create_rest_resource(apigateway_client):
```

Comment:
I'm curious about how to create this fixture. Deleting the API container should remove most of the resources created under path, templates, etc - backend integration lingers tho. Any particular use case where this didn't happen? meaning deleting the API wasn't removing resources?

Reply:
I'm deleting it, it was me not exactly knowing how it would behave. Thanks!